src/content/en/updates/2019/08/native-file-system.md

```diff
@@ -0,0 +1,457 @@
+  with `inPlace: true`).
+* Writing to a file using a [`WritableStream`][writablestream].
+* The [`FileSystemDirectoryHandle.resolve()`][fs-dir-handle] method.
+
+## Security and permissions {: #security-considerations }
+
+<style>
+  figcaption {font-size: smaller;}
+</style>
+
+We’ve designed and implemented the Native File System using the core
+principles we defined in
+[Controlling Access to Powerful Web Platform Features][powerful-apis],
+including user control and transparency, and user ergonomics.
+
+### Opening a file or saving a new file
+
+<figure class="attempt-right">
+  <a href="/web/updates/images/2019/08/fs-open.jpg">
+    <img src="/web/updates/images/2019/08/fs-open.jpg">
+  </a>
+  <figcaption>
+    File picker used to open an existing file for reading.
+  </figcaption>
+</figure>
+
+When opening a file, the user provides permission to read a file or
+directory via the file picker. The open file picker can only be shown via
+a user gesture when served from a [secure context][secure-contexts]. If
+the user changes their mind, they can cancel the selection in the file
+picker and the site does not get access to anything. This is the same
+behavior as the `<input type="file">` element.
+
+<div class="clearfix"></div>
+
+<figure class="attempt-left">
+  <a href="/web/updates/images/2019/08/fs-save.jpg">
+    <img src="/web/updates/images/2019/08/fs-save.jpg">
+  </a>
+  <figcaption>
+    File picker used to save a file to disk.
+  </figcaption>
+</figure>
+
+Similarly, when a web app wants to save a new file, the browser will show
+the save file picker, allowing the user to specify the name and location
+of the new file. Since they are saving a new file to the device (versus
+overwriting an existing file), the file picker grants the app permission
+to write to the file.
+
+<div class="clearfix"></div>
+
+#### Restricted folders
+
+To help protect users and their data, the browser may limit the user’s
+ability to  save to certain folders, for example, core operating system
+folders like Windows, the macOS Library folders, etc. When this happens,
+the browser will show a modal prompt and ask the user to choose a
+different folder.
+
+### Modifying an existing file or directory
+
+A web app cannot modify a file on disk without getting explicit permission
+from the user.
+
+#### Permission prompt
+
+<figure class="attempt-right">
+  <a href="/web/updates/images/2019/08/fs-save-permission.jpg">
+    <img src="/web/updates/images/2019/08/fs-save-permission-crop.jpg">
+  </a>
+  <figcaption>
+    Prompt shown to users before the browser is granted write
+    permission on an existing file.
+  </figcaption>
+</figure>
+
+If a person wants to save changes to a file that they previously granted
+read access, the browser will show a modal permission prompt, requesting
+permission for the site to write changes to disk. The permission request
+can only be triggered by a user gesture, for example, clicking a “Save”
```

Comment:
Nit: before you just wrote Save and Save As without the quotes, now you write "Save"